docs/source/tutorials/stallion.ipynb

```diff
@@ -1917,7 +1917,7 @@
    "source": [
     "# calcualte metric by which to display\n",
     "predictions = best_tft.predict(val_dataloader, return_y=True)\n",
-    "mean_losses = SMAPE(reduction=\"none\")(predictions.output, predictions.y).mean(1)\n",
+    "mean_losses = SMAPE(reduction=\"none\").loss(predictions.output, predictions.y[0]).mean(1)\n",
```

Comment:
this is a workaround for the noteboo, but not a fix for #1614, right?

Reply:
It's a workaround.